docs/src/user/manual/data-upload/panel.md

```diff
@@ -137,13 +137,13 @@ The `DIGITISE` button is a powerful tool that allows the user to digitise parcel
 
 6. **Digitised parcel**: Digitised parcel is shown.
 
-7. **Save boundary**: After the user has successfully digitised and outlined the parcel boundary using the above options, they can finalise the process by clicking the `SAVE BOUNDARY` button. This saves the digitised boundary and incorporates it into the user's property's data.
+7. **Save boundary**: After the user has successfully digitised and outlined the parcel boundary using the above options, they can finalise the process by clicking the `SAVE BOUNDARY` button. This saves the digitised boundary and incorporates it into the user's property data.
```

Comment:
This is for the specific property so ownership is needed for both user and property
```suggestion
7. **Save boundary**: After the user has successfully digitised and outlined the parcel boundary using the above options, they can finalise the process by clicking the `SAVE BOUNDARY` button. This saves the digitised boundary and incorporates it into the user's property's data.
```

Reply:
okay